### PR TITLE
Several updates to UGM page

### PR DESCRIPTION
**Pull request by jennaswa**
This PR makes several updates to the UGM 2024 page on the website, including:

- Add UGM and TYC logos
- Add updated agenda (name/title additions/corrections)
- Add abstracts book
- Add logistical details (i.e., transportation and food suggestions)

### CODE REVIEW

**Review by micaela-matta — APPROVED** [review]

File: public/data/mdaUGM2024-abstracts.pdf
Comment (orbeckst) [inline public/data/mdaUGM2024-abstracts.pdf:1]: I would call the "Verbal" presentations **Oral Presentations** and Abstracts (as opposed to Posters).

**Review by orbeckst — APPROVED** [review]
Great, thank you. See one comment on abstracts booklet.

**Comment by jennaswa** [comment]
Thanks, @orbeckst! To clarify, do you mean “Oral Presentations” and “Poster
Presentations” as the two categories?

*Jenna Swarthout Goddard *| *PhD*
***@***.*** | + 49 (160) 244 0829

**Comment by orbeckst** [comment]
I only meant to replace "verbal" with "oral" where it occurred, nothing more than that.

File: pages/ugm2024.md [hunk 1]
```diff
@@ -54,11 +65,32 @@ The conference will start at 9:00 am on Wednesday, 21 August and will close arou
...
 
+Most places in London are easily accessible by public [buses](https://tfl.gov.uk/modes/buses/) and the [underground/rail/tram](https://tfl.gov.uk/tube-dlr-overground/status/) systems. It is cheapest to pay for transportation using contactless and Oyster cards; read more about [transportation fares here](https://tfl.gov.uk/fares/).
+
 Citizens of several countries may travel to the United Kingdom to attend a meeting without obtaining a visa. Please visit the [gov.uk](https://www.gov.uk/browse/visas-immigration/tourist-short-stay-visas) website for more information about travel visas. Attendees requiring a letter clarifying the purpose of travel should [contact the MDAnalysis project manager][email]. 
 
+## Meals
+Food and beverages will be provided where noted on the [agenda](https://www.mdanalysis.org/pages/ugm2024/#program) (i.e., Coffee Break, Lunch, Reception). For mealtimes that are not listed, we encourage you to explore Lisbon to check out England's cuisine options! We have also compiled a few suggestions below.
```
Comment (micaela-matta) [inline pages/ugm2024.md:78]: just seen the Lisbon/London typo!

**Comment by jennaswa** [comment]
Thanks, @orbeckst and @micaela-matta! Made a couple of direct commits to the master to update the abstracts book and fix the typo.

**Pull request merged**